src/TicketingSystem.Messaging/Producer/KafkaProducerProvider.cs

```diff
@@ -0,0 +1,40 @@
+﻿using com.ticketingSystem;
+using Confluent.Kafka;
+using Confluent.SchemaRegistry.Serdes;
+
+namespace TicketingSystem.Messaging.Producer
+{
+    public class KafkaProducerProvider : IProducerProvider
+    {
+        private bool _disposed;
+
+        public IProducer<string, MessageValue> Producer { get; }
+
+        public KafkaProducerProvider(KafkaConfigurationProvider config)
+        {
+            var avroSerializerConfig = new AvroSerializerConfig
```

Comment:
Looks, like you use Json serializer, so you can remove all Avro-related code from your project. Or I'm missing something?